elf2flt.c

```diff
@@ -2036,5 +2037,5 @@ int main(int argc, char *argv[])
  * c-basic-offset: 4
  * tab-width: 8
  * end:
- * vi: tabstop=8 shiftwidth=4 textwidth=79 noexpandtab
+ * vi: tabstop=8 shiftwidth=4 noexpandtab
```

Comment:
unrelated change ?

Reply:
Oops. Do you want me to force push a new version with this removed?

Reply:
yeah, rewrite the commits so there are only the two "clean" ones

Reply:
Done!